ipatests/azure/azure-run-tests.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -x
```

Comment:
The script does not set ``set -e`` to abort and fail the script when one of the commands fail.

Reply:
Fixed.